src/administration/web.md

```diff
@@ -1,6 +1,6 @@
 # Web Interface
 
-[Platform.sh](https://platform.sh/) provides a responsive Web Interface which allows you to interact with your projects and manage your environments. Everything you can do with this interface you can also achieve with the  [CLI (Command Line Interface)](/gettingstarted/cli.md) and vice versa.
+Platform.sh provides a responsive Web Interface which allows you to interact with your projects and manage your environments. Everything you can do with this interface you can also achieve with the  [CLI (Command Line Interface)](/gettingstarted/cli.md) and vice versa.
```

Comment:
Why remove the link here?

Reply:
My edit was based on point 3 in the section **Language usage** in the Contributing Guidelines. 

> The name of the company is always written as "Platform.sh", not simply "Platform". It should not be linked anywhere.

I noticed after my commit Platform.sh linked on other pages, so I wonder if this is a new standard that should be taken out of the Contributing Guidelines?

Reply:
Or perhaps replaced with:

> The name of the company is always written as "Platform.sh", not simply "Platform", It should not be linked anywhere, unless it is the first mention on a page.

Reply:
> The name of the company is always written as "Platform.sh", not simply "Platform". It should not be linked anywhere, unless it is the first mention on a page.

Reply:
Oh!  Hm.  I did write that, didn't I... :smile: 

It's more likely that we just didn't clear up a lot of the old links lying around.  We should probably not link the name, although I MIGHT be convinced that having the first mention on the very first page be a link is a worthwhile exception (as I apparently thought it was, wrongly), since otherwise there's no link to our actual website from the docs.

Which now that I think about it is also wrong, because we have the header bar attached anyway and that does have a link back to the main site.  So yeah, kill all the links!